ieee754r/fabsd32.c

```diff
@@ -1,11 +1,10 @@
 /* Calculate the absolute value for a _Decimal32 type
 
-   Copyright (C) 2006 IBM Corporation.
-   Copyright (C) 2007-2015 Free Software Foundation, Inc.
+   Copyright (C) 2007-2019 Free Software Foundation, Inc.
```

Comment:
This should get updated to 2020. Otherwise, LGTM if you squash the changes.

Reply:
Done! https://github.com/libdfp/libdfp/blob/42de205d01ce4542cc3a4a94a6b439f7bdf43840/ieee754r/fabsd32.c#L3